### PR TITLE
TRT-1754: Enable a few backend sampling for observer

### PR DESCRIPTION
**Pull request by xueqzhan**

### CODE REVIEW

**Comment by xueqzhan** [comment]
pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-multiarch-master-nightly-4.17-ocp-e2e-aws-ovn-arm64

**Comment by xueqzhan** [comment]
/pj-rehearse periodic-ci-openshift-release-master-ci-4.17-upgrade-from-stable-4.16-e2e-aws-ovn-upgrade

File: ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh [hunk 1]
```diff
@@ -61,6 +61,6 @@ then
 fi
 
 openshift-tests run-resourcewatch > "${ARTIFACT_DIR}/run-resourcewatch.log" 2>&1 &
-DISABLED_MONITOR_TESTS="apiserver-availability,apiserver-new-disruption-invariant,disruption-summary-serializer,external-service-availability,incluster-disruption-serializer,image-registry-availability,ingress-availability,pod-network-avalibility,service-type-load-balancer-availability"
+DISABLED_MONITOR_TESTS="apiserver-new-disruption-invariant,disruption-summary-serializer,incluster-disruption-serializer,pod-network-avalibility"
```
Comment (p0lyn0mial) [inline ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh:64]: thanks for the pr!

@xueqzhan do you know why `apiserver-new-disruption-invariant` is disabled ?

@xueqzhan also, do  you know the difference between `apiserver-new-disruption-invariant` and `apiserver-availability` and `external-service-availability` ?

**Pull request merged**